readme.rst

```diff
@@ -1,23 +1,51 @@
+Launchpad Manual
+==============================
+
+Welcome to the official Launchpad Documentation Manual. This manual is designed
```

Comment:
The name is "Launchpad Manual", not "Launchpad Documentation Manual".

Reply:
fixed